src/debian/01_password

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/sh
+
+# Set grub password : "grub-mkpasswd-pbkdf2 -c 65536 -s 256 -l 64" and replace $GRUB_PASS
+# Default password : toor
+
+GRUB_PASS=grub.pbkdf2.sha512.65536.5CFD987412BE492E5B25B524DA4FF8687841D62DD31F4CCE0608796204FAF7CF741D90B6DAB3F547C9625D995786F054836A060D45F96E64642CF029E2AF0503E030A8712DC71315F4C64E0D42CDC292CE6D8FBE63575EA4B1111AD42D61A89A406BAC02B47BA9C6BE9C7781E3ECBA3C506CC9B6020C3B808C85689687DB70C6E245E380A33F9ACF197FE9BD3C8FDA30588C429DBE4CD0FA022229C42E605C5DAF6E654AE0E160864DA2B6FDAA35247D88C886A876F402C2DDB56612F469A28F72551FD97C061E9630DCB8655A072DE0BA761D576F110C04E65DC3AE6EC6A62A89AA683934FEAE50EA0DF9BF6595CA4C9A3C1A1C11A657CCC40359CF7C6EB3E7.82DB321EBD4A971F73F49B4BEA96A42C47809C8E0E08D605A59C3460ECAF07ED199401A368EAA3787702F33A09795F5C6AC9CA12219229F890C16535F9A0544F
+
+echo superusers="root"
+echo password_pbkdf2 root $GRUB_PASS
+awk "/^menuentry 'Debian/,/}/" /boot/grub/grub.cfg | sed "s/menuentry.*{/menuentry 'SEAPATH' --unrestricted {/g"
```

Comment:
I think this could belong in build_debian_iso. This awk/sed patch is tricky and could fail if grub.cfg is not exactly as you want. Check if it wouldn't be more convenient and robust in build_debian_iso.

Reply:
Use only sed, okey in this repo, Rename seapath